File: code/backend/douban-service/src/test/java/com/ilife/douban/entity/Book.java
package com.ilife.douban.entity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Table(name = "BOOK")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Book {
    @Id
    @Column(name = "ID")
    private String id;
    @Column(name = "NAME")
    private String name;
    @Column(name = "AUTHOR")
    private String author;
    @Column(name = "PRICE")
    private String price;
    @Column(name = "RANKING")
    private float ranking;
    @Column(name = "HOT")
    private Integer hot;
}

File: code/crawllers/douban-python-crawller/entity/Book.py
class Book:
    def __init__(self, _id, name, author, price, ranking, hot):
        self.id = _id
        self.name = name
        self.author = author
        self.price = price
        self.ranking = ranking
        self.hot = hot

    def __str__(self):
        result = self.id + " " + self.name + " " + self.author + " " + self.price + " " + self.ranking + " " + self.hot
        return result
